packages/ts-moose-lib/src/consumption-apis/runner.ts

```diff
@@ -252,6 +252,130 @@ const apiHandler = async (
   };
 };
 
```

Comment:
`createMainRouter()` wraps the pre-existing `apiHandler()` and tries to route to a `WebApp` first if possible, falling back to consumption `Api` and ultimately a 404 if none found.